include/freerdp/window.h

```diff
@@ -188,13 +188,13 @@ struct _WINDOW_STATE_ORDER
 	UINT32 ownerWindowId;
 	UINT32 style;
 	UINT32 extendedStyle;
-	UINT32 showState;
+	UINT8 showState;
 	RAIL_UNICODE_STRING titleInfo;
 	INT32 clientOffsetX;
 	INT32 clientOffsetY;
 	UINT32 clientAreaWidth;
 	UINT32 clientAreaHeight;
-	UINT32 RPContent;
+	UINT8 RPContent;
```

Comment:
While this is the correct value range size type it will break public API, so keep the old one.